README.md

```diff
@@ -1,2 +1,27 @@
-# Capstone-project_Concert
+# CAPSTONE PROJECT
```

Comment:
- [ ] Kindly include in the video pieces of code you are interested in as in the project requirement.
![image](https://user-images.githubusercontent.com/24703171/168912138-2944da83-e03b-45df-b0ae-18c307db4658.png)

Reply:
ok